content/en/docs/bundles/distro-full.md

```diff
@@ -0,0 +1,37 @@
+---
+title: The distro-full bundle
+linkTitle: distro-full
+description: "Kubernetes distribution, full stack version"
+weight: 20
+---
+
+This is a Cozystack platform configuration intended for use as a Kubernetes distribution, designed for installation on Talos Linux.
+
+This allows you to use Cozystack as a reliable Kubernetes distribution for bare metal.
```

Comment:
Not only is it possible on bare metal, it is also possible on virtual machines, but here it is harder to say that it is mainly sharpened under bare metal, although it can be installed on virtual machines.

Reply:
Thank you for suggestion, fixed!